src/main/java/Task.java

```diff
@@ -0,0 +1,23 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public void markAsDone(){
+        isDone = true;
+    }
+
+    public void markAsUndone(){
+        isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
```

Comment:
This could include the brackets so that you dont need to put the brackets on line 53, eg: isDone ? "[X]" : "[  ]"